src/models/StakeRequest.ts

```diff
@@ -0,0 +1,79 @@
+import { Address, Bytes32 } from '../types';
+
+export default class StakeRequest {
+  private _id: string;
+
+  private _amount: string;
```

Comment:
Please see the updated code. These properties are already defined in model/repository classes.
We might not need this class.